server/api/attendance/attendance.integration.js

```diff
@@ -0,0 +1,122 @@
+'use strict';
+
+import app from '../..';
+import Attendance from './attendance.model';
+import User from '../user/user.model';
+import SmallGroup from '../smallgroup/smallgroup.model';
+import request from 'supertest';
+import superagent from 'superagent';
+import {seed} from '../../config/seed';
+import {getSessionFor} from '../../auth/util.integration';
+
+describe('Attendance API:', () => {
+
+  describe("Small group attendance", () => {
+
+    before(seed);
+
+    var mentorSession, studentSession;
+
+    before(() => getSessionFor('mentor').then(session => mentorSession = session));
+    before(() => getSessionFor('student').then(session => studentSession = session));
+
+    let smallgroup, mentor, student;
+
+    before(() => {
+      // Add the mentor to a small group
+      return Promise.all([
```

Comment:
im not sure how else to do parallel promises haha

sidenote: multiple assignment and async/await makes this block a lot cleaner